packages/react-router/lib/router/router.ts

```diff
@@ -3551,7 +3556,7 @@ export function createStaticHandler(
     request: Parameters<StaticHandler["query"]>[0],
     {
       requestContext,
-      filterMatchesToLoad,
+      filterMatchesToLoad: filterMatchesToLoadBase,
```

Comment:
This can be reverted now I think?

```suggestion
      filterMatchesToLoad,
```